entry/src/block_component.rs

```diff
@@ -1328,7 +1365,7 @@ mod tests {
     fn create_parent_ready_update() -> UpdateParentV1 {
         UpdateParentV1 {
             new_parent_slot: 42,
-            new_parent_block_id: Hash::default(),
+            new_parent_block_id: Hash::new_from_array([42u8; 32]),
```

Comment:
might be nice to make this something that's not 42.

Just on the off chance we introduce some parsing bug, seems more likely we would catch it if value is different from `new_parent_slot`